backoffice/providers/auth.provider.ts

```diff
@@ -32,4 +32,12 @@ export class AuthProvider extends BaseAuthProvider {
   private isAdmin(user: UserDto) {
     return user.role === ROLES.ADMIN;
   }
+
+  override async handleLogout({res}: {res: Response}) {
+    // Remove auth cookies
+    res.setHeader('Set-Cookie', [
+      `backoffice=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly`,
```

Comment:
Wouldn't work if we just remove the cookie/cookies altogether instead of rewriting this value?